pkgs/applications/virtualization/kvmtool/default.nix

```diff
@@ -1,12 +1,13 @@
-{ stdenv, fetchzip, lib, dtc }:
+{ stdenv, fetchgit, lib, dtc }:
 
 stdenv.mkDerivation {
   pname = "kvmtool";
-  version = "unstable-2023-07-12";
+  version = "unstable-2024-06-16";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
specifically version must start with a number

Reply:
This repository does not have any releases, tags or refs, so I'm taking snapshot. What do you think? What version string should I specify?

Reply:
Maybe 2024-06-16-snapshot?

Reply:
If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning / tagging schema - then the latest upstream version in the above schema should be 0

Reply:
Okay, I fixed that.